### ISSUE TITLE
the parameter  is different with the tensorflow

### ISSUE THREAD

**Issue by ivylinden** [Simon4Yan/eSPGAN#5]
Hi, thanks for your work, I find the parameter is different with the tensorflow, such as the learning rate  ,tne batchsize and the  crop_width

**Comment by Simon4Yan** [comment]
These are data processing settings. You could modify them if you would like to. I use them since I want to decrease the training time (I do not have enough GPUs).
